xwiki-platform-core/xwiki-platform-search/xwiki-platform-search-solr/xwiki-platform-search-solr-ui/src/main/resources/Main/SolrSearch.xml

```diff
@@ -796,8 +759,31 @@ a.search-result-highlightAll:after {
   padding: .1em .2em;
 }
 
-.search-facet.expanded .search-facet-header {
-  background-image: url("$doc.getAttachmentURL('dark-grey-disclosure-arrow-down.png')");
+.search-facet .search-facet-header .facet-toggler, button.facet-value-toggler {
+  background: transparent;
+  float: right;
+}
+
+button.facet-value-toggler:not(:active) {
+  color: $theme.btn-primary-color;
+}
+
+.search-facet .search-facet-header .facet-toggler:active, button.facet-value-toggler:active {
+  box-shadow: unset;
+}
+
+.search-facet .search-facet-header .facet-toggler span, button.facet-value-toggler span {
+  transform: rotate(90deg);
```

Comment:
Have you tried this with the silk icon theme? Not every icon theme uses a span as container for the icon.

Reply:
I did not try this out.
Addressed in https://github.com/xwiki/xwiki-platform/pull/2311/commits/8706ef95cb59c12842618d1a2b2d257d5a7d89f5 :+1: